### DISCUSSION TITLE
Synchronize with non-root OneDrive folder

### DISCUSSION THREAD

**Discussion by CarlosRuiz-globalqss** [abraunegg/onedrive#1459] [Q&A] [answered]
By default I see the application synchronize against the root folder of OneDrive

It would be nice if in configuration file I could select a folder to synchronize.

So, it would be the same as
sync_dir = "~/OneDrive"
but for the remote side, something like:
remote_sync_dir = "OneDrive"

Probably a similar result could be achieved with skip_dir, skipping all the folders that I don't want to synchronize, but still is possible that I create a folder on mobile that I don't want to synchronize to my laptop and it will be automatically synchronized by default.

Regards,

Carlos Ruiz

**Comment by abraunegg** [comment, marked as answer]
@CarlosRuiz-globalqss 
Please read the documentation as this *is* possible. Investigate:
* 'sync_list'
* `--single-directory`

Refer to: https://github.com/abraunegg/onedrive/blob/master/docs/USAGE.md

**Comment by flavio92ux** [comment]
Hi. Dont work. I want that create a new directory in my remote and not copy the new files for the root folder Ondrive.

**Reply by abraunegg** [reply]
@flavio92ux 
Rather than hijacking an old thread, and providing almost zero referential information ... I cant help you as to *why* something is not working.

Start a new discussion post please. provide the output of `--display-config`, and clearly articulate *what* you are trying to do.
